github_actions/lib/dependabot/github_actions/file_updater.rb

```diff
@@ -65,17 +65,37 @@ def updated_workflow_file_content(file)
             gsub(/@.*+/, "@#{new_req.fetch(:source).fetch(:ref)}")
 
           # Replace the old declaration that's preceded by a non-word character
-          # and followed by a whitespace character (comments) or EOL
+          # and followed by a whitespace character (comments) or EOL.
+          # If the declaration is followed by a comment, attempt to update
+          # any version comments associated with SHA source refs.
```

Comment:
What do you think about tweaking this slightly?

Something like:
```
# If the declaration is followed by a comment that lists the version associated
# with the SHA source ref, then update the comment to the human-readable new version.
# However, if the comment has additional text beyond the version, for safety
# we skip updating it in case it's a custom note, todo, warning etc of some kind.
```